### PR TITLE
Attribute detail card

### PR DESCRIPTION
**Pull request by vlichtenberg**

### CODE REVIEW

File: ui/app/main/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@openremote/core": "^1.1.0",
+    "@openremote/or-attribute-card": "^1.0.0",
```
Comment (richturner) [inline ui/app/main/package.json:19]: Please set version number to match other components (i.e. `1.1.0`)

File: ui/app/main/src/components/pages/page-insights.ts [hunk 2]
```diff
@@ -53,8 +110,8 @@ const viewerConfig: DataViewerConfig = {
             gridRowStart: "1"
         },
         defaults: [{
-            assetId:"5442Ite2XVp3Ky9X2y647a",
-            attributes: ["UVIndex"]
+            assetId:"4m4dWZUAV4IepHKc8Wwv85",
```
Comment (richturner) [inline ui/app/main/src/components/pages/page-insights.ts:113]: Not clear what these defaults mean but no good having `assetIds` hardcoded as they won't be valid between installations

File: ui/component/or-attribute-card/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# @openremote/or-attribute-card  \<or-attribute-card\>
+[![NPM Version][npm-image]][npm-url]
+[![Linux Build][travis-image]][travis-url]
+[![Test Coverage][coveralls-image]][coveralls-url]
+
+Web Component for displaying an input control.
```
Comment (richturner) [inline ui/component/or-attribute-card/README.md:6]: Description of what this component does

File: ui/component/or-attribute-card/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Install
+```bash
+npm i @openremote/or-input
```
Comment (richturner) [inline ui/component/or-attribute-card/README.md:10]: Invalid

File: ui/component/or-attribute-card/package.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@openremote/or-attribute-card",
+  "version": "1.0.0",
```
Comment (richturner) [inline ui/component/or-attribute-card/package.json:3]: As mentioned change the version number

File: ui/component/or-attribute-card/src/index.ts [hunk 6]
```diff
@@ -0,0 +1,441 @@
+import {css, customElement, html, LitElement, property, PropertyValues, query, unsafeCSS} from "lit-element";
+import {classMap} from "lit-html/directives/class-map";
+import i18next from "i18next";
+import {Asset, DatapointInterval, MetaItemType, ValueDatapoint} from "@openremote/model";
+import {manager, DefaultColor4, DefaultColor5} from "@openremote/core";
+import Chart, {ChartTooltipCallback} from "chart.js";
+import {getContentWithMenuTemplate} from "@openremote/or-chart";
+import {InputType, OrInputChangedEvent} from "@openremote/or-input";
+import {OrAttributeHistoryEvent} from "@openremote/or-attribute-history";
+import {getMetaValue} from "@openremote/core/dist/util";
```
Comment (richturner) [inline ui/component/or-attribute-card/src/index.ts:10]: Make sure or-package references are correctly listed in `tsconfig.json` and `package.json`

File: ui/component/or-attribute-card/src/index.ts [hunk 7]
```diff
@@ -0,0 +1,441 @@
...
+        const response = await manager.rest.api.AssetResource.queryAssets({
+            ids: [id],
+            recursive: false
+        });
+
+        if (response.status !== 200 || !response.data) {
+            return {};
+        }
+
+        return response.data[0];
```
Comment (richturner) [inline ui/component/or-attribute-card/src/index.ts:281]: Maybe check you don't have an empty array

File: ui/component/or-attribute-card/src/index.ts [hunk 8]
```diff
@@ -0,0 +1,441 @@
...
+                value: "week"
+            },
+            {
+                text: i18next.t(DatapointInterval.MONTH),
+                value: "month"
+            },
+            {
+                text: i18next.t(DatapointInterval.YEAR),
+                value: "year"
+            }
```
Comment (richturner) [inline ui/component/or-attribute-card/src/index.ts:408]: You can just pass value through `i18next.t()` at the point at which these options are consumed so just return desired `DatapointInterval` options here

File: ui/component/or-attribute-card/README.md (hunk 4 above)
Comment (vlichtenberg) [reply ui/component/or-attribute-card/README.md]: Woops...

File: ui/component/or-attribute-card/src/index.ts (hunk 8 above)
Comment (vlichtenberg) [reply ui/component/or-attribute-card/src/index.ts]: I've pushed a solution, hope it's what you meant!

File: ui/app/main/src/components/pages/page-insights.ts (hunk 2 above)
Comment (vlichtenberg) [reply ui/app/main/src/components/pages/page-insights.ts:113]: As discussed I've passed this to Don.

File: ui/component/or-attribute-history/package-lock.json [hunk 9]
```diff
@@ -0,0 +1,143 @@
+{
```
Comment (richturner) [inline ui/component/or-attribute-history/package-lock.json:1]: Please delete, should be using `yarn`

**Comment by vlichtenberg** [comment]
> Once insights page default behaviour is figured out and `default` removed from config this can be merged

I've asked @DonWillems what we should do with this and if he has an idea, if he could file it using Git's ticketing system.

**Pull request merged**